ivy.xml

```diff
@@ -38,23 +36,12 @@
         <dependency org="org.aspectj" name="aspectjrt" rev="1.8.10"/>
         <dependency org="org.json" name="json" rev="20200518"/>
 
-        <dependency org="mil.army.usace.hec" name="cwms-db-jooq" rev="9.3.5" conf="cwms-api->default">
-            <exclude org="org.jooq.pro"/>					
-        </dependency>
-        <dependency org="mil.army.usace.hec" name="cwms-db-aspects" rev="9.3.5" conf="cwms-api->default">
-            <exclude org="org.jooq.pro"/>					
-        </dependency>
-        <dependency org="mil.army.usace.hec" name="cwms-db-dao" rev="9.3.5">
-            <exclude org="org.jooq.pro"/>					
-        </dependency>        
-
-        <dependency org="mil.army.usace.hec" name="hec-db-core" rev="9.3.5">
```

Comment:
I'm not sure how these hec-db-core and hec-db-jdbc dependencies got added in, but since they weren't needed at compile-time I removed them.

Reply:
that was me flailing around. Hence why I asked for help.